envoy/Makefile

```diff
@@ -19,7 +19,7 @@ ENVOY_BINS = $(ENVOY_BIN) ./bazel-bin/cilium_integration_test
 CHECK_FORMAT ?= ./bazel-bin/check_format.py.runfiles/envoy/tools/check_format.py
 
 BAZEL ?= bazel
-BAZEL_OPTS ?= --batch
```

Comment:
Well need to keep the `--batch` option for the PKG_BUILD case.

Reply:
Sure, I'll update for this.